### PR TITLE
Feature: Adding Frequently Asked Questions screen on mobile app

### PR DESCRIPTION
**Pull request by Omar-Belghaouti**
# Description
This PR is dedicated to add the FAQ screen to the mobile app.

## Type of change

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: mobile/src/redux/reducers/faq-screen/faq.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import { FAQData } from ".";
+
+export const faqData: FAQData = [
```
Comment (ZibanPirate) [inline mobile/src/redux/reducers/faq-screen/faq.ts:3]: i'm thinking of moving this entire file under `./common/src/web` so we can use it on both here and in `./web` as well

File: mobile/src/screens/faq/index.tsx [hunk 2]
```diff
@@ -1,16 +1,50 @@
...
+          <View key={`category-${index}`}>
+            <Text style={faqStyles.title}>{title}</Text>
+            <List.Section>
+              {questions.map(({ question, answer }, index) => (
+                <List.Accordion key={`question-${index}`} title={question}>
+                  <Markdown
+                    style={{
+                      body: faqStyles.description,
+                    }}
+                    onLinkPress={(url) => {
```
Comment (ZibanPirate) [inline mobile/src/screens/faq/index.tsx:30]: we may wanna abstract this logic just like we did with the `LinkV2` in `./web`, we can maybe write e function that does the checking, and then just pass it to the `onLinkPress` both here and in `Article` Screen

**Review by ZibanPirate — APPROVED** [review]
Code reads ok, tested it locally and works great (maybe missing some margins), thanks @Omar-Belghaouti  !

I left two comments, feel free to either merge this PR, then we do a follow-up PR to address them, or just do it here before merging

File: mobile/src/redux/reducers/faq-screen/faq.ts (hunk 1 above)
Comment (Omar-Belghaouti) [reply mobile/src/redux/reducers/faq-screen/faq.ts:3]: @ZibanPirate okay we'll do that, but the links that begins with `/` must be treated as well to navigate to them.

File: mobile/src/screens/faq/index.tsx (hunk 2 above)
Comment (Omar-Belghaouti) [reply mobile/src/screens/faq/index.tsx:30]: Good idea, we'll do this on this PR before merging.

**Pull request merged**